modules/nw-sriov-network-operator.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="nw-sriov-network-operator_{context}"]
+
+= Configuring an SR-IOV Network Operator for virtual functions
+The Single Root I/O Virtualization (SR-IOV) network operator is used to allocate and configure Virtual Functions (VFs) from SR-IOV-supporting Physical Function NICs on the nodes.
+
+For more information on deploying the operator, see _Installing the SR-IOV Network Operator_.
```

Comment:
You could capitalise "Operator". For more information, see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-capitalization.

Reply:
Done.